phpcs.xml

```diff
@@ -1,8 +1,9 @@
 <?xml version="1.0"?>
 <ruleset name="WordPress ActivityPub">
 	<description>WordPress ActivityPub Standards</description>
-	<file>.</file>
 
+	<file>.</file>
+	<exclude-pattern>.(git|github|vscode|idea|wordpress-org)</exclude-pattern>
 	<exclude-pattern>*\.(inc|css|js|svg)</exclude-pattern>
```

Comment:
Could these be merged?

Reply:
I am not sure that this is a good idea. The first line ignores dotfiles, the second one file extensions.